sunpy/tests/figure_tests_env_3.5.yml

```diff
@@ -17,13 +17,10 @@ dependencies:
 - fontconfig=2.12.1=2
 - freetype=2.5.5=2
 - glib=2.50.2=1
-- gst-plugins-base=1.8.0=0
-- gstreamer=1.8.0=0
```

Comment:
I had to get rid of these to make the environment install, but the images generated fine without them.

Reply:
It would be nice if we could unpin as many items as possible in the future.